api/python/provisioner/inputs.py

```diff
@@ -819,9 +822,6 @@ class NetworkParams():
     primary_data_netmask: str = ParamGroupInputBase._attr_ib(
         _param_group, descr="primary node data interface netmask"
     )
-    primary_data_network_iface: List = ParamGroupInputBase._attr_ib(
```

Comment:
Remove `iface` in the hope that we're no longer using it and it was not found in `params_spec` as well. Do we need to include it?

Reply:
It should be renamed to data_public_interfaces and data_private_interfaces,
and related changes to be accompanied